src/controls/IRoomControllerV2.ts

```diff
@@ -0,0 +1,380 @@
+import { CurrentStateValue, OldStateValue } from '../main';
+import { Control } from '../structure-file';
+import { ControlBase, ControlType } from './control-base';
+export class IRoomControllerV2 extends ControlBase {
+    private uuid = '';
+
+    async loadAsync(type: ControlType, uuid: string, control: Control): Promise<void> {
+        this.uuid = uuid;
+
+        await this.updateObjectAsync(uuid, {
+            type: type,
+            common: {
+                name: control.name,
+                role: 'thermo',
+            },
+            native: { control: control as any },
+        });
+
```

Comment:
For forwards compatibility, we should always call `loadOtherControlStatesAsync()` from `loadAsync()` after updating the root object. See other control handlers.
If Loxone ever adds states, they will automatically be present in the object tree.

Reply:
I don't see this is necessary as there aren't any other states outside of those specifically created.